modules/iam-service-account/iam.tf

```diff
@@ -101,6 +108,17 @@ resource "google_project_iam_member" "project-roles" {
   member  = local.resource_iam_email
 }
 
+resource "google_service_account_iam_member" "cicd-resman" {
```

Comment:
Should probably use a different name for this resource

Reply:
it works ok for a resource name, what would you suggest instead?

Reply:
it's ok under the fast folder, but this is the SA module. Isn't `cicd-resman` out of place here?

Reply:
it's the SA *resource*

Reply:
:facepalm: